### ISSUE TITLE
Lovelace Glance card not distributing width correctly

### ISSUE THREAD

**Issue by Devqon** [MCrissDev/Ariela#2]
When using the glance card with 3 entities, it does not render the items with equal width. It looks like it always assumes 4 items.
![image](https://user-images.githubusercontent.com/9316480/52407515-d2d13180-2ad0-11e9-82f5-b4ee4d1ed66f.png)

**Comment by MCrissDev** [comment]
Hello,
Can you share a screenshot of the issue? Thank you.

**Comment by Devqon** [comment]
@MCrissDev I added a screenshot. I created the issue on my mobile, and was surprised that I cannot upload an image from Android.

**Comment by MCrissDev** [comment]
Now i see the problem. Thank you, i will fix it on the next release.

**Comment by MCrissDev** [comment]
One more question, columns did you set to 3 in your glance card config?

**Comment by Devqon** [comment]
Yes, I also tried to explicitly set the number of columns:
```
- type: glance
  show_state: true
  columns: 3
  entities:
    - entity: media_player.laptop
    - entity: light.woonkamer_tv
    - entity: light.woonkamer_bank
```

**Comment by MCrissDev** [comment]
Got it. Thank you.

**Comment by MCrissDev** [comment]
Fixed in version 1.2.9.1
